### PR TITLE
TW-92 Add page on Deployment Control - Enterprise app

### PR DESCRIPTION
**Pull request by claireriley**
[Spec](https://postmanlabs.atlassian.net/l/c/nVvLJA7J)

Waiting on Postman screenshot (see comment)

### CODE REVIEW

File: src/pages/docs/administration/managing-enterprise-deployment.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+It is always recommended to [run silent installations with logging enabled](#enabling-verbose-logging-for-installation).
+
+### Enabling verbose logging for installation
+
+The `msiexec` tool can be configured to output debug log information about the installation process with the `/l*v` option. For example, you can output debug information to `C:\log.txt`:
+
+```
+msiexec /i path\to\package.msi /l*v C:\log.txt
+```
+
```
Comment (numaanashraf) [inline src/pages/docs/administration/managing-enterprise-deployment.md:90]: @jviotti Are there any instructions or steps we can share to inform users how to update the Enterprise App? Updating is done by installing a newer version of the application right? cc @claireriley

**Review by numaanashraf — CHANGES_REQUESTED** [review]

File: src/pages/docs/administration/managing-enterprise-deployment.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    * [Enabling verbose logging for installation](#enabling-verbose-logging-for-installation)
+
+* [Uninstalling the Postman Enterprise app](#uninstalling-the-postman-enterprise-app)
+
+## Downloading the Postman Enterprise app
+
+You must be a [Postman team admin](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) to access the Postman Enterprise app package.
+
+To download, navigate to Postman and select **Team** in the upper right > **Team Settings**
```
Comment (jkonrath-postman) [inline src/pages/docs/administration/managing-enterprise-deployment.md:37]: missing period at end

**Review by jkonrath-postman — APPROVED** [review]
Approved, once you get the screenshots and everything Numaan mentioned.

File: src/pages/docs/administration/managing-enterprise-deployment.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+<!-- need screenshot -->
+
+## Installing the Postman Enterprise app
+
+Once you've downloaded the Postman Enterprise MSI package, you can move forward with installing the app.
+
+### INSTALLDIR
+
+The `INSTALLDIR` public property is used to select a custom installation. If this public property is not manually overwritten, it defaults to `%PROGRAMFILES%\[company]\[name]` for system-wide installations and `%USERPROFILE%\AppData\Local\Programs\[company]\[name]` for per-user installations.
```
Comment (jviotti) [inline src/pages/docs/administration/managing-enterprise-deployment.md:47]: In this case, `[company]` is always "Postman", so we can be clear there

Comment (jviotti) [reply src/pages/docs/administration/managing-enterprise-deployment.md:47]: This is also always about the Postman Enterprise app, so we can also inline the name. @numaanashraf was is the final name we are going for?

File: src/pages/docs/administration/managing-enterprise-deployment.md (hunk 1 above)
Comment (jviotti) [reply src/pages/docs/administration/managing-enterprise-deployment.md:90]: @numaanashraf I'll extend the Starship docs accordingly, but the summary is the following:

- Upgrading the app is done by simply installing the new version of the MSI (the Windows Installer system will realize it is an upgrade out of the box)
- It is crucial that users upgrade by passing the exact same public properties they used at installation time. For example, if the user first installed by passing `INSTALLDIR=C:\custom` and `MSIINSTALLPERUSER=1`, then the new MSI has to be installed with the exact same properties
- We must clarify that downgrading the version of the app is not supported. If the user attempts to downgrade, an error message will be shown. The user can force-downgrade by manually removing the current version and then installing the older version

File: src/pages/docs/administration/managing-enterprise-deployment.md (hunk 2 above)
Comment (mark-anthony-postman) [reply src/pages/docs/administration/managing-enterprise-deployment.md:37]: Maybe better to bring the navigation path together?
"To download, navigate to Postman and select **Team > Team Settings** in the upper right."

**Review by mark-anthony-postman — APPROVED** [review]
Made one minor suggestion, otherwise LGTM 🙂

**Comment by numaanashraf** [comment]
@claireriley Here's the download UI screenshot

download-enterprise-app

File: src/pages/docs/administration/managing-enterprise-deployment.md (hunk 3 above)
Comment (numaanashraf) [reply src/pages/docs/administration/managing-enterprise-deployment.md:47]: @claireriley @jviotti Yep. Company will be "Postman" and name will be "Postman Enterprise"

Comment (claireriley) [reply src/pages/docs/administration/managing-enterprise-deployment.md:47]: @jviotti @numaanashraf will the name "Postman Enterprise" have a hyphen or space?

Comment (numaanashraf) [reply src/pages/docs/administration/managing-enterprise-deployment.md:47]: @claireriley No hyphen. It has a space.

**Review by numaanashraf — APPROVED** [review]

**Pull request merged**